includes/class-wc-install.php

```diff
@@ -1195,7 +1195,7 @@ public static function plugin_row_meta( $links, $file ) {
 			$row_meta = array(
 				'docs'    => '<a href="' . esc_url( apply_filters( 'woocommerce_docs_url', 'https://docs.woocommerce.com/documentation/plugins/woocommerce/' ) ) . '" aria-label="' . esc_attr__( 'View WooCommerce documentation', 'woocommerce' ) . '">' . esc_html__( 'Docs', 'woocommerce' ) . '</a>',
 				'apidocs' => '<a href="' . esc_url( apply_filters( 'woocommerce_apidocs_url', 'https://docs.woocommerce.com/wc-apidocs/' ) ) . '" aria-label="' . esc_attr__( 'View WooCommerce API docs', 'woocommerce' ) . '">' . esc_html__( 'API docs', 'woocommerce' ) . '</a>',
-				'support' => '<a href="' . esc_url( apply_filters( 'woocommerce_support_url', 'https://woocommerce.com/my-account/tickets/' ) ) . '" aria-label="' . esc_attr__( 'Visit premium customer support', 'woocommerce' ) . '">' . esc_html__( 'Premium support', 'woocommerce' ) . '</a>',
+				'support' => '<a target="_blank" href="' . esc_url( apply_filters( 'woocommerce_support_url', 'https://github.com/ClassicPress-research/classic-commerce/issues' ) ) . '" aria-label="' . esc_attr__( 'Visit issues  support section on github', 'woocommerce' ) . '">' . esc_html__( 'Support', 'woocommerce' ) . '</a>',
```

Comment:
All links that have `target="_blank"` should also have `rel="noopener noreferrer"`

More info: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

I think we can trust GitHub not to engage in `window.opener` shenanigans, but this is still a guideline that should always be followed.

Reply:
Thanks for the reminder.